proposals/lip-0045.md

```diff
@@ -134,6 +134,7 @@ In this section, we specify the substores that are part of the Interoperability
 | `CHAIN_ID_MAINCHAIN` | bytes | 0x00000000 | Chain ID of the Lisk mainchain in the mainnet. |
 | `CHAIN_NAME_MAINCHAIN` | string | "lisk-mainchain" | Name of the Lisk mainchain. |
 | **Interoperability Store** | | | |
+| `STORE_PREFIX_INTEROPERABILITY` | bytes | 0x83ed0d25 | Prefix of the interoperability store. |
```

Comment:
This corresponds to the integer 2213350693. What is the reason for this specific choice?

Reply:
The rationale is explained in LIP40. We want to separate the interop store from other stores to minimize the witness length in CCUs.